vue-components/.storybook/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	overrides: [
+		{
+			files: [ 'preview.js' ],
```

Comment:
I'm again a little confused by the blurred lines between run time and build time in storybook. If I'm not mistaken, `preview.(j|t)s` is actually executed at run time, so I'd rather remove this `.eslintrc.js` entirely, and just rename `preview.js` -> `preview.ts` and not bother specifying additional rules for run time `.js` files.